docs/user_guide/storing/docindex.md

```diff
@@ -136,6 +199,8 @@ need to have compatible schemas.
     - A and B have the same field names and field types
     - A and B have the same field names, and, for every field, the type of B is a subclass of the type of A
 
+    In particular this means that you can easily [index predefined Documents](#using-a-predefined-document-as-schema) into a Document Index.
```

Comment:
```suggestion
    In particular, this means that you can easily [index predefined Documents](#using-a-predefined-document-as-schema) into a Document Index.
```

Reply:
What's the policy on capitalizing `Document` now that we don't use that class name? I think @samsja mentioned on Discord we don't do that any more.

Reply:
I think we should still capitalize it, since it is a concept in our library. Lowercased it looks a bit weird and "unofficial" to me. Plus, I think the rule of thumb was always that "concepts" are capitalized, whereas classes go in between `backticks`

Reply:
No strong feeling here. But tehcnically speaking Document is not a concept in term of code in the library

Reply:
I think it is a concept but just not a class, otherwise "concept" and "class" would be synonyms. But I just checked the pydantic documentation, they don't capitalize "model". So no strong feeling either